lib/beaker/hypervisor/docker.rb

```diff
@@ -224,6 +224,7 @@ def provision
           :password => root_password,
           :port => port,
           :forward_agent => forward_ssh_agent,
+          :keys_only => 'yes',
```

Comment:
Thanks for your contribution @b4ldr. It seems to me that we are using username/password to authenticate the SSH connection, so disabling looking through the ssh-agent for keys makes sense. My worry is that if anyone is using ssh-agent to connect, this would break it. Another solution would be to set the "auth_methods" to change the order and try the password method first, like so:

```suggestion
          :auth_methods => [“password”, “publickey”, “hostbased”, “keyboard-interactive”]
```

from https://net-ssh.github.io/ssh/v1/chapter-2.html

Reply:
Sounds good to me, updated